src/app/[locale]/page.tsx

```diff
@@ -1,15 +1,17 @@
-import { useTranslations } from "next-intl";
-import Link from "next/link";
-import Navbar from "../components/Navbar";
+import Link from 'next/link';
 
-const Home = () => {
-  const t = useTranslations("Index");
+import Wrapper from '../components/Wrapper';
+import Navbar from '../components/Navbar';
+import Header from '../components/Header';
+import Services from '../components/Services';
 
+const Home = () => {
   return (
-    <div>
+    <Wrapper>
```

Comment:
what is the use of this wrapper component and why you couldn't add the div of it directly here instead of creating a component for it?

Reply:
Two reasons for that Wrapper! First one was for the implementation of the blue background. This wrapper was helping me to center the content of the page, but at the same time letting the box of the blue blackground go behind the limits of the wrapper. Second reason is because wanted the page.tsx file as clean as possible from tailwind, or anything else. So I felt was a good practice!

Reply:
totally right I didn't realize this advantage to keep a simple page.tsx. Maybe you can rename it to have a more descriptive name like BodyWrapper for example, what do you think?